### DISCUSSION TITLE
My hubs deployment isn't live after 24 hours after deploying my stack

### DISCUSSION THREAD

**Discussion by ybinstock** [mozilla/hubs#3278] [Hubs Cloud Questions] [answered]
I've followed the instructions from the YouTube video to deploy my stack and it seemed to go successfully but when I go to

https://virtualcelebrations.io, I get 502 error.

On the Cloud Formation stage, this is what I see. See attached
<img width="1291" alt="Screen Shot 2020-11-05 at 6 22 43 PM" src="https://user-images.githubusercontent.com/8645066/98274863-40570e00-1f94-11eb-8132-b9f6ab5d29cc.png">

I don't think I did anything wrong in the deployment process, so I don't know what's causing this issue. 

Any help would be greatly appreciated.

**Comment by robinkwilson** [comment, marked as answer]
Can you go through the steps listed in my answer here? : https://github.com/mozilla/hubs/discussions/3071

First try updating the stack, then terminate instances to restart the system. Let me know if it works!

**Reply by robinkwilson** [reply]
(Click "Mark as unread" when you reply! Thanks)
